src/DotLiquid.Tests/ConditionTests.cs

```diff
@@ -695,6 +699,80 @@ public void TestEqualOperatorsWorksOnEnum()
             AssertEvaluatesFalse("enum", "!=", "'Yes'");
         }
 
+        [Test]
+        public void TestBlankObject()
+        {
+            _context = new Context(CultureInfo.InvariantCulture);
+            _context["dict"] = new Dictionary<string, string> { { "abc", "xyz" } };

```

Comment:
might benefit from using full English words for readability - i.e., dictionary and array.